File: SingleAPIExample/python/Get.py
import splitit

username, password, api_key = (
    "username",
    "password",
    "api_key",
)

api_client = splitit.ApiClient(splitit.Configuration(username, password, api_key, sandbox=True))


def get_installment_plan(ipn):
    resp = api_client.InstallmentPlanApi.installment_plan_get(
        splitit.GetInstallmentsPlanSearchCriteriaRequest(
            splitit.InstallmentPlanQueryCriteria(
                installment_plan_number=ipn,
            )
        )
    )
    return resp

File: SingleAPIExample/java/Get.java
import com.splitit.*;
import com.splitit.sdk.model.*;
import java.math.BigDecimal;


public class Get {
    private String apiKey, username, password;
    public Get(String username, String password, String apiKey){
        this.username = username;
        this.password = password;
        this.apiKey = apiKey;
    }

    public void example(String ipn) throws ApiException{
        ApiClient apiClient = new ApiClient()
            .sandbox(true)
            .username(username)
            .password(password)
            .apiKey(apiKey);


        var result = apiClient.getInstallmentPlanApi().installmentPlanGet(
            new GetInstallmentsPlanSearchCriteriaRequest()
            .queryCriteria(
                new InstallmentPlanQueryCriteria()
                .installmentPlanNumber(ipn)
            )
        );
        System.out.println(result);
    }
}
